src/containers/my-cooperations/cooperations-container/CooperationContainer.tsx

```diff
@@ -62,7 +65,21 @@ const CooperationContainer: React.FC<CooperationContainerProps> = ({
   const cooperationTable = (
     <EnhancedTable
       columns={columnsToShow}
-      data={{ items }}
+      data={{
+        items: items.map((item) => {
+          const roleBasedStatus =
+            item.needAction.role === userRole
+              ? StatusEnum.NeedAction
+              : StatusEnum.RequestToClose
+          return {
```

Comment:
Add empty space before return, please
```suggestion
         
          return {
```